tools/docker/builder/openwrt/scripts/build-openwrt.sh

```diff
@@ -12,8 +12,17 @@ mkdir -p files/etc
 printf '%s=%s\n' "OPENWRT_REPOSITORY" "$OPENWRT_REPOSITORY" >> files/etc/prplwrt-version
 printf '%s=%s\n' "OPENWRT_VERSION" "$OPENWRT_VERSION" >> files/etc/prplwrt-version
 if [ "$TARGET_PROFILE" = DEVICE_NETGEAR_RAX40 ] ; then
-    ./scripts/gen_config.py intel_mips
-    cat profiles/intel_mips.yml >> files/etc/prplwrt-version
+    # Add prplmesh to the list of packages of the profile:
+    sed -i 's/packages:/packages:\n  - prplmesh-dwpal/g' profiles/intel_mips.yml
+    yq write --inplace profiles/netgear_rax40.yml feeds -f profiles_feeds/netgear-rax40.yml
+    ./scripts/gen_config.py netgear_rax40 debug
+    cat profiles_feeds/netgear-rax40.yml >> files/etc/prplwrt-version
+elif [ "$TARGET_PROFILE" = DEVICE_AX6000_2000_ETH_11AXUCI ]; then
+    # Add prplmesh to the list of packages of the profile:
+    sed -i 's/packages:/packages:\n  - prplmesh-dwpal/g' profiles/axepoint.yml
+    yq write --inplace profiles/axepoint.yml feeds -f profiles_feeds/netgear-rax40.yml
```

Comment:
It's a bit strange to update the axepoint profile with netgear-rax40.yml.... Maybe we should call it prplmesh.yml.

Why did you split the axepoint and the rax40 again, BTW? AFAIU it *is* possible to build both together in one shot, and I don't see any reason not to do so. Or is it a problem to retrieve the correct image in the later stages?

Reply:
I originally wanted to use the intel_mips profile to indeed build both at the same time - but @rmelotte said it will be easier for CI if we build them separately, probably due to the reason you raised.

Reply:
It actually makes sense - AxePoint and netgear-rax40 are almost identical SW wise, so keeping it named netgear-rax40.yml looks OK to me. We could change it in the future though...